strings.txt

```diff
@@ -393,10 +393,10 @@ PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_AFTER
 	DE	Nach den Titeln
 	NL	Na afspeellijst
 
-PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_HIDDEN
-	EN	Do not show
-	DE	Nicht Anzeigen
-	NL	Niet tonen
+PLUGIN_QOBUZ_WORK_PLAYLIST_POSITION_INTEGRATED
+	EN	Integrate with work headings
+	DE	Mit Werktiteln integrieren
+	NL	integreren met werktitels
```

Comment:
Should probably be upper case?

Reply:
I'll do this - for all languages?

Reply:
I was only referring to the Dutch `integreren`, sorry.